eclipse-collections/src/main/java/org/eclipse/collections/impl/map/mutable/primitive/ObjectBooleanHashMap.java

```diff
@@ -1109,6 +1109,14 @@ private int maxOccupiedWithData()
         return Math.min(capacity - 1, capacity / OCCUPIED_DATA_RATIO);
     }
 
+    /**
+     * Rehashes every element in the set into a new backing table of the smallest possible size and eliminating removed sentinels.
+     */
+    public void compact()
```

Comment:
I'd suggest the name `trimToSize()` for consistency with UnifiedMap and ArrayList.